redis/src/main/scala/zio/redis/codecs/CRC16.scala

```diff
@@ -1,4 +1,4 @@
-package zio.redis.codec
+package zio.redis.codecs
```

Comment:
This change is not really related to the PR scope, but I like it so let it stay.

Reply:
yeah you're right, sorry about that. i renamed it because at certain point i had a collision of codec package and codec method on redis package object, and i forgot about it. i'll keep it then.